### ISSUE TITLE
cb is causing issues

### ISSUE THREAD

**Issue by RazvanDH** [kss-node/kss-node#263]
Since version `2.2.0`, you removed `cb = cb || function() {};` fallback for `cb` and it returns an error when trying to generate the styleguide:

```
TypeError: cb is not a function
    at kssHandlebarsGenerator.generate (/Users/someprojectiworkon/kss/generator/handlebars/kss_handlebars_generator.js:268:3)
    at /Users/someprojectiworkon/gulp/styleguide.js:89:30
    at parse (/Users/someprojectiworkon/node_modules/kss/lib/parse.js:90:3)
    at /Users/someprojectiworkon/node_modules/kss/lib/traverse.js:123:13
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:380:3)
```

**Comment by JohnAlbin** [comment]
In kss-node 2.1.x, kssHandlebarsGenerator.generate() did _not_ have a cb parameter.

In 2.2.x, I added callbacks and default functions to kss_generator.js. But I failed to add the default functions to kss_handlebars_generator.js.

Fixing…

**Comment by JohnAlbin** [comment]
@RazvanDH I rolled a new release with the fix: 2.3.4

**Comment by RazvanDH** [comment]
thank you for the quick fix, @JohnAlbin !
